src/lib/libhtml5_webgl.js

```diff
@@ -109,7 +109,7 @@ var LibraryHtml5WebGL = {
 #endif
 
     var canvas = findCanvasEventTarget(target);
-
+    if (canvas?.canvas) { canvas = canvas.canvas; }
```

Comment:
This is worth a comment I think.

And, if this is for offscreen canvas (as the PR title suggests) then can it be behind an ifdef, so we do not increase code size for most users?